presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestAllDatatypesFromHiveConnector.java

```diff
@@ -315,4 +351,59 @@ public void testSelectAllDatatypesParquetFile()
                         true,
                         "kot binarny".getBytes()));
     }
+
+    private static void populateDataToHiveTable(String tableName)
+    {
+        onHive().executeQuery(format("INSERT INTO %s VALUES(" +
```

Comment:
Then the `testInsertIntoSelectToHiveTableAllHiveSimpleTypes` test is pointless if I cover it in this method. I don't have strong opinions. Can change that.